pulsar-client/src/main/java/org/apache/pulsar/client/impl/BatchMessageIdImpl.java

```diff
@@ -105,23 +105,11 @@ public byte[] toByteArray() {
         return toByteArray(batchIndex, batchSize);
     }
 
-    public boolean ackIndividual() {
-        return acker.ackIndividual(batchIndex);
-    }
-
     public boolean ackCumulative() {
         return acker.ackCumulative(batchIndex);
     }
 
-    public int getOutstandingAcksInSameBatch() {
-        return acker.getOutstandingAcks();
-    }
-
     public int getBatchSize() {
-        return acker.getBatchSize();
-    }
-
```

Comment:
These are the APIs exposed by the client, can they be deleted directly without any compatibility?